Source/Shaders/ModelExperimental/ModelExperimentalVS.glsl

```diff
@@ -42,12 +42,12 @@ void main()
     FeatureIds featureIds;
     featureIdStage(featureIds, attributes);
 
-    #ifdef HAS_SELECTED_FEATURE_ID
+   /* #ifdef HAS_SELECTED_FEATURE_ID
     SelectedFeature feature;
     selectedFeatureIdStage(feature, featureIds);
     // Handle any show properties that come from the style.
     cpuStylingStage(attributes.positionMC, feature);
-    #endif
+    #endif*/
```

Comment:
I'm guessing this was temporarily commented out for debugging? Remember to uncomment when done.